src/views/mfa-verify/PushForm.js

```diff
@@ -49,6 +63,19 @@ define(['okta'], function (Okta) {
         factorName: this.model.get('factorLabel'),
         deviceName: this.model.get('deviceName')
       });
+
+      var that = this;
+      if (that.enabled) {
+        window.setTimeout(function () {
```

Comment:
Yep:
1. Put this logic in a postRender
2. Just call the function that submit calls when it's clicked, i.e. there's no reason to simulate a DOM event when we ourselves control the handler that is triggered when the event happens

i.e.
```javascript
postRender: function () {
  if (featureEnabledAndCookiebla) {
    this.submit();
  }
}
```